_team/spark.md

```diff
@@ -0,0 +1,11 @@
+---
+firstname: "Seongbin"
+lastname: "Park"
+group: "member"
+title: "Developer & Instructor"

```

Comment:
One quick note here: for people with two roles, we usually use:

```
title: "Developer"
secondary: "Instructor"
```

Do you mind making that quick change? Other than that, it looks totally good to me!